editoast/src/modelsv2/mod.rs

```diff
@@ -157,7 +157,7 @@ mod tests {
             }
         }
 
-        #[derive(Clone, ModelV2)]
```

Comment:
You may also add this bound to `modelsv2::trait Model`

Reply:
Done in ff1e7dd06.